components/dialog/stories/template.js

```diff
@@ -24,65 +24,86 @@ export const Template = ({
 	footer = [],
 	customClasses = [],
 	id = getRandomId("dialog"),
+	size = "medium",
+	layout,
+	hasHeroImage = false,
+	heroImageUrl,
+	customStyles = {},
 } = {}, context = {}) => {
-	const { globals = {}, updateArgs } = context;
-	const scale = globals.scale ?? "medium";
-	const toggleOpen = function () {
-		updateArgs({ isOpen: !isOpen });
-	};
+	const { updateArgs } = context;
+	const toggleOpen = () => updateArgs({ isOpen: !isOpen });
 
 	const Dialog = html`
 		<div
 			class=${classMap({
 				[rootClass]: true,
-				[`${rootClass}--${scale}`]: true,
-				[`${rootClass}--dismissable`]: isDismissable && !isFullScreen && !isFullScreenTakeover,
-				[`${rootClass}--fullscreen`]: isFullScreen,
-				[`${rootClass}--fullscreenTakeover`]: isFullScreenTakeover,
+				[`${rootClass}--dismissable`]: isDismissible && (layout !== "fullscreen" || layout !== "fullscreenTakeover"),
```

Comment:
Both ways are perfect, just wanted to share a nice shorthand I like to use sometimes leveraging the array and the `every` or `some` functions to make updates easier (just add new values to the array to update the logic).

```suggestion
				[`${rootClass}--dismissable`]: isDismissible && ["fullscreen", "fullscreenTakeover"].every(l => layout !== l),
```

Reply:
yesssss